lib/ai/use-meeting-baas-chat.ts

```
@@ -0,0 +1,136 @@
+    }
+    
+    // Debug API key again right before sending
+    if (typeof window !== 'undefined') {
+      const apiKey = localStorage.getItem('meetingbaas_api_key') || localStorage.getItem('auth_token');
+      console.log('API key before submit:', apiKey ? `exists (${apiKey.length} chars)` : 'missing');
+    }
+    
+    setStatus('submitted');
+    
+    try {
+      console.log('Calling engine.prompt with input:', input.slice(0, 20) + '...');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)